docs/preview/02-Features/powershell/azure-management.md

```diff
@@ -0,0 +1,65 @@
+
+---
+title: "Azure Management"
+layout: default
+---
+
+# Azure Management
```

Comment:
Clean and to-the-point as always, 👍 Great job!!

Reply:
Thanks!